fastlane_core/lib/fastlane_core/update_checker/changelog.rb

```diff
@@ -4,18 +4,22 @@ module FastlaneCore
   class Changelog
     class << self
       def show_changes(gem_name, current_version)
+        did_show_changelog = false
+
         self.releases(gem_name).each_with_index do |release, index|
           next unless Gem::Version.new(release['tag_name']) > Gem::Version.new(current_version)
           puts ""
           puts release['name'].green
           puts release['body']
+          did_show_changelog = true
 
           next unless index == 2
           puts ""
           puts "To see all new releases, open https://github.com/fastlane/#{gem_name}/releases".green
           break
         end
-        puts "\nUpdate using 'sudo gem update #{gem_name.downcase}'".green
+
+        puts "\nPlease update using `#{UpdateChecker.update_command(gem_name: gem_name)}`".green if did_show_changelog
```

Comment:
So I can do the following? 
```ruby
puts "", "Please update using `#{UpdateChecker.update_command(gem_name..."
```
I feel like this is more confusing, especially for non-Ruby developers, as you have to know how Ruby handles this. `\n` is universally understood by developers.